lib/PuppeteerSharp/Page.cs

```diff
@@ -714,8 +716,33 @@ private void HandleException(string exceptionDetails)
         {
         }
 
-        private void OnDialog(MessageEventArgs e)
+        private void OnDialog(PageJavascriptDialogOpeningMessageResponse message)
         {
+
+            DialogType dialogType;
+            if (message.Type == "alert")
```

Comment:
maybe create a dictionary to map the type to its string representation to clean this up?